models/BaseConfig.cfc

```diff
@@ -1012,6 +1012,14 @@ component accessors="true" {
 		}
 
 		var thisEventGatewayInstances=getEventGatewayInstances() ?: [];
+		var i=0;
```

Comment:
I assume saving the same instance twice wasn't overwriting?

Reply:
Ideed. The gateway instances is "just an array", so you could end up with two gateway instances with the same type. Coldfusion does not like that.